### PR TITLE
JAMES-2598 Packaging Cassandra-RabbitMQ-Ldap produt with deb/rpm

### PR DESCRIPTION
**Pull request by hoangdat**

### CODE REVIEW

**Comment by hoangdat** [comment]
#1949

File: dockerfiles/packaging/guice/cassandra/package.sh [hunk 1]
```diff
@@ -45,19 +45,31 @@ docker run \
    --volume $PWD/dockerfiles/run/guice/cassandra-rabbitmq/destination:/cassandra-rabbitmq/destination \
    -t james/project -s $SHA1
 
+# Compile James with Cassandra + RabbitMQ + Swift + Ldap
+docker run \
+   --rm \
+   --volume PWD/.m2:/root/.m2 \
+   --volume $PWD:/origin \
+   --volume $PWD/dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination:/cassandra-rabbitmq-ldap/destination \
```
[the remark this answers is not in the document]
Comment (hoangdat) [reply dockerfiles/packaging/guice/cassandra/package.sh]: Please check my new modification

[the remark this answers is not in the document]
Comment (chibenwa) [reply dockerfiles/packaging/guice/cassandra/package.sh]: looks good to me

[the remark this answers is not in the document]
Comment (chibenwa) [reply dockerfiles/packaging/guice/cassandra/package.sh]: `# Compile James with Cassandra`

The commant should rather be `# Compile James`

**Comment by hoangdat** [comment]
Build triggered. sha1 is original commit.

**Comment by hoangdat** [comment]
Build started sha1 is original commit.

**Comment by hoangdat** [comment]
Build finished. No test results found.

**Comment by hoangdat** [comment]
Build triggered. sha1 is original commit.

**Comment by hoangdat** [comment]
Build started sha1 is original commit.

**Comment by chibenwa** [comment]
`MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated` test this please

**Comment by chibenwa** [comment]
`MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated`

test this please

**Comment by chibenwa** [comment]
```
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR] Failures: 
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR]   ManageableMailQueueContract.getSizeShouldReturnMessageCountWhenSeveralMails:71 expected:<[2]L> but was:<[3]L>
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR] Errors: 
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR]   MailQueueContract.concurrentEnqueueDequeueShouldNotFail:387 » NotTerminated
```

test this please

**Comment by trantienduchn** [comment]
```
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR] Errors: 
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR]   SwiftTempAuthObjectStorageBlobsDAOBuilderTest.builtBlobsDAOCanStoreAndRetrieve:94 » Timeout
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [INFO] 
[1e46e9b7868ebe971c0253f46abb7c38dc6dec0e] [ERROR] Tests run: 124, Failures: 0, Errors: 1, Skipped: 0
```
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**